requirements.txt

```diff
@@ -6,5 +6,5 @@ redis
 graphviz
 requests
 python-socketio
--e git+https://github.com/RedisGraph/redisgraph-py#egg=redisgraph
--e git+https://github.com/http-apis/hydra-python-core@master#egg=hydra_python_core
+redisgraph
```

Comment:
We can't move ahead with the latest release of `redisgraph`. They have backwards incompatible change which breaks the agent. Currently we are depending upon the last working commit for us of redisgraph.(see requirements.txt of develop branch of agent). 

cc: @Mec-iS

Reply:
this should be written and noted somewhere. There is a comment or a note in README? We should have a PR open to fix it, it could be a good task for early contributors.